crds/jwst/tpns/nircam_apcorr.tpn

```diff
@@ -19,4 +19,4 @@ PUPIL                      C   C   R    CLEAR,FLAT,F162M,F164N,GDHS0,GDHS60,MASK
 
 EEFRACTION                 C   R   (is_imaging_mode(EXP_TYPE))   0.0:1.0
 
-APCORR                     C   R   R   1.0:1.8e308
+APCORR                     C   A   R   ndarray
```

Comment:
If you're interested in retaining the range check, something like this should work:

```
EXT1   D   X   R   ((EXT1_ARRAY.DATA["APCORR"]>=1.0).all())
```

Reply:
Good to know. For this particular case, nothing along these lines have been specified, so will leave off in favor of the next comment.